docs/providers/Serviceplatformen.md

```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
+Serviceplatformen primarily exhibits personal data. It is the responsibility of the municipality to ensure that this data is processed in accordance with 
```

Comment:
We should probably state that serviceplatform is only for use by Municipalities or systems acting on their behalf.

Reply:
This text was copy-pasted from their page :D